frontend/components/vault/VaultLiquidationTransactionFlow.stories.js

```diff
@@ -0,0 +1,52 @@
+import { storiesOf } from '@storybook/vue';
+import faker from 'faker';
+import VaultLiquidationTransactionFlow from '~/components/vault/VaultLiquidationTransactionFlow';
+import {
+    generateFakeVaultLiquidatedTransaction,
+    generateFakeVaultNotLiquidatedTransaction,
+} from '~/helpers/generateFakeVault';
+
+const common = {
+    components: { VaultLiquidationTransactionFlow },
+    data: () => ({
+        vaultTransaction: generateFakeVaultNotLiquidatedTransaction(),
+        network: 'mainnet',
```

Comment:
The network is also provided inside `vaultTransaction`, so no need to pass it as a separate prop